src/main/java/org/candlepin/controller/refresher/RefreshWorker.java

```diff
@@ -68,48 +62,37 @@
  * update their local representations.
  */
 public class RefreshWorker {
-    private static Logger log = LoggerFactory.getLogger(RefreshWorker.class);
+    private static final Logger log = LoggerFactory.getLogger(RefreshWorker.class);
 
     /**
-     * The number of times to retry certain CRUD operations which may fail as a result of
-     * entity versioning constraints
+     * The number of times to retry the refresh operation if it fails as a result of a constraint
+     * violation.
      */
-    private static final int VERSIONING_CONSTRAINT_VIOLATION_RETRIES = 4;
-
-    /** Default grace period for orphaned entities */
-    private static final int ORPHANED_ENTITY_DEFAULT_GRACE_PERIOD = -1;
```

Comment:
With this change, what happens when a manifest consumer's entitlement expires, and subsequently the corresponding pool is deleted during the manifest import, and no other pool is using the product/content that this pool provided? The products/content stay in the database because they are in the global namespace?

Reply:
Correct. Refresh/import will no longer delete products or contents for any reason (again).

I was skeptical of that at first thought, but given that (a) hosted disables orphan cleanup entirely and (b) sat wants a long lead time on it as well (30 days from detection + next import IIRC), it seems like something users don't actually want anyway. Given the substantial reduction in data usage, I don't expect anyone will actually care; and if they do, we can revisit this in a branch specifically for Sat.